tools/prow-auto-bumper/parser.go

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// prow-auto-bumper finds stable Prow components version used by k8s,
+// and creates PRs updating them in knative/test-infra
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"math"
+	"strings"
+	"time"
+
+	"github.com/knative/test-infra/shared/ghutil"
+)
+
+// Helper method for adding a newly discovered tag into pv
+func (pv *PRVersions) getIndex(image, tag string) int {
```

Comment:
move the methods in the same file with the definition of PRVersions

Reply:
done